index.html

```diff
@@ -0,0 +1,80 @@
+        </div>
+        <div class="form-group">
+            <label class="email-label">Email</label></br>
+            <input type="text" class="email " name="Email" placeholder="Enter your Email" required>
+        </div>
+        <div class="form-group">
+            <label class="number-label">Age(optional)</label></br>
+            <input type="number" class="age" name="Age" min="10" max="99" placeholder="Age">
+        </div>
+        <div class="form-group">
+            <p>which option bests describe your current role?</p>
+            <select class="dropdown">
+            <option value="Select current role">Select current role</option>
+            <option value="Students">Student</option>
+            <option value="Full Time Job">Full Time Job</option>
+            <option value="Full Time Learner">Full Time Learner</option>
+            <option value="Prefer not to say">Prefer not to say</option>
+            <option value="Other">Other</option>
+            </select>
+        </div>
+        <div class="form-group">
+            <p>Would you recommend freeCodeCamp to a friend?</p>
+            <input type="radio" name="freeCodeCamp" value="Definitely" >Definitely</br>
+            <input type="radio" name="freeCodeCamp" value="Maybe">Maybe</br>
+            <input type="radio" name="freeCodeCamp" value="Not sure">Not sure</br>
+        </div>
+        <div class="form-group">
+            <p>What is your favourite feature in freeCodeCamp?</p>
+            <select class="dropdown">
+            <option value="select an option">select an option</option>
+            <option value="Challenges">Challenges</option>
+            <option value="Projects">Projects</option>
+            <option value="Community">Community</option>
+            <option value="Open Source">Open Source</option>
+            </select>
+        </div>
+        <div class="form-group">
+            <p>What would you like to see improved? (check all that apply)</p>
+            <input type="checkbox" value="Front-end Projects">Front-end Projects</br>
+            <input type="checkbox" value="Back-end Projects">Back-end Projects</br>
+            <input type="checkbox" value="Data Visualization">Data Visualization</br>
+            <input type="checkbox" value="Challenges">Challenges</br>
+            <input type="checkbox" value="Open Source Community">Open Source Community</br>
+            <input type="checkbox" value="Gitter help rooms">Gitter help rooms</br>
+            <input type="checkbox" value="Videos">Videos</br>
+            <input type="checkbox" value="City Meetups">City Meetups</br>
+            <input type="checkbox" value="Wiki">Wiki</br>
+            <input type="checkbox" value="Forum">Forum</br>
+            <input type="checkbox" value="Additional Courses">Additional Courses</br>                                                                                                                
```

Comment:
- Try to avoid using the `<br>` tag instead use the CSS property for line space. And `<br>` tag is self enclosing tag, that's why you are getting a Stylelint error. You can read more about the `<br>` tag here [Mozila](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br)